executor/runtime/docker/docker.go

```diff
@@ -2047,13 +2047,18 @@ func (r *DockerRuntime) handleDockerEvent(message events.Message, statusMessageC
 		l.Debugf("Skipping docker start event for %s, no need to update the pod", cName)
 		return nonTerminalDockerEvent
 	case "die":
-		// TODO: Handle the difference between platform/user sidecar, not all
-		// "die"s should result in a Finish
 		if exitCode := message.Actor.Attributes["exitCode"]; exitCode == "0" {
 			statusMessageChan <- runtimeTypes.StatusMessage{
 				Status: runtimeTypes.StatusFinished,
 				Msg:    cName + " container successfully exited with 0",
 			}
+		} else if exitCode := message.Actor.Attributes["exitCode"]; exitCode == "137" && cName != mainContainerName {
```

Comment:
Nit: why not `if message.Actor.Attributes["exitCode"] == "137"`?

Reply:
Huh. Yea I guess since we are not using the `exitCode` var later, we don't need it.